homeassistant/components/vizio/__init__.py

```diff
@@ -17,24 +19,25 @@
 )
 
 
-def validate_auth(config):
+def validate_auth(config: ConfigType) -> ConfigType:
     """Validate presence of CONF_ACCESS_TOKEN when CONF_DEVICE_CLASS=tv."""
     token = config.get(CONF_ACCESS_TOKEN)
     if config[CONF_DEVICE_CLASS] == "tv" and not token:
         raise vol.Invalid(
             f"When '{CONF_DEVICE_CLASS}' is 'tv' then '{CONF_ACCESS_TOKEN}' is required.",
             path=[CONF_ACCESS_TOKEN],
         )
+
     return config
 
 
 VIZIO_SCHEMA = {
-    vol.Required(CONF_HOST): cv.string,
-    vol.Optional(CONF_ACCESS_TOKEN): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Required(CONF_HOST): cv.string,
```

Comment:
Please revert this change.

Reply:
Done but can I ask why? I had moved them around the match the order I plan to show them in the config flow form so that it would be easier to keep the config schema and config flow schemas in sync

Reply:
Normally we start with required items and regardless we shouldn't mix clean ups with other changes.

Reply:
Oh got it, makes sense, thanks!